src/server/publicApi.js

```diff
@@ -1,125 +1,181 @@
-import cors from '@koa/cors';
-import Koa from 'koa';
-import koaBody from 'koa-body';
-import Router from 'koa-router';
-import request from 'superagent';
-import { ElevationOfPrivilege } from '../game/eop';
-import { API_PORT, INTERNAL_API_PORT } from '../utils/constants';
-import { getTypeString } from '../utils/utils';
+import cors from "@koa/cors";
```

Comment:
Is there a special reason why you converted this to double quotes? The other files use single quotes and we should keep it for consistency...

Reply:
No, this was just the result of my editor reformatting the file, I'll change it back